x/reward/types/action_builder.go

```diff
@@ -228,6 +230,13 @@ func (v *VoteActionBuilder) GetEventCriteria() *EventCriteria {
 	})
 }
 
+var govVoteMsgUrls = []string{
```

Comment:
I recommend moving govVoteMsgUrls into the AddEvent method since nothing else uses it.

Reply:
Having it in `AddEvent` means it has to be recreated every single time it's needed. Those calls to `sdk.MsgTypeURL` aren't just simple look-ups. But it probably is better to set it the first time `AddEvent` is called because of `MsgTypeURL`'s possible reliance on reflection and/or proto registration. So I left the variable scoped to the package, but set it if it hasn't been set yet.

Reply:
I like the new approach, and it makes sense from a performance perspective.